docs/guides/10-integrating-react.md

```diff
@@ -0,0 +1,183 @@
+This guide will lead you through the process of integrating Airship in [React.js](https://reactjs.org/).
+
+>  We are working on React bindings that offer a better developer experience in the future.
+
+## Including Styles
+
+Install `airship-style` package and 
+
+ ```
+npm i @carto/airship-style
+```
+
+Load the styles into your code
+
+```css
+@import '@carto/airship-style/dist/airship.css';
+```
+
+## Web components
+
+A web component is just an HTML tag with some attributes that control its behaviour. Simply include the HTML tag, and edit its properties through attributes or through javascript as you would do with a normal HTML element.
+
+To render a web component simply include its tag inside the REACT render function:
+
+
+```jsx
+render() {
+  <as-infowindow src="https://thecatsite.com/styles/thecatsite/xenforo/sources/notice3.png"></as-infowindow>
+}
+```
+
+
+## Loading web components
+
+Install `airship-components` package 
+
+```
+npm i @carto/airship-components
+```
+
+In the entry point of your application call `defineCustomElements` pasing `window` as a parameter. Once you do this this, all airship components
+will be available.
+
+```js
+import { defineCustomElements } from '@carto/airship-components';
+defineCustomElements(window);
+```
+
+In order to manage the state, either listening to events or synchronizing the attributes we got two options.
+
+## Option 1: Manage component logic in the parent layer.
+
+The simplest dirty-hack is to update all attributes, and re-render everything. Even so,  [content attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#Content_versus_IDL_attributes) only accept strings as parameters forcing to set the element properties from javascript.
+
+
+Use the **Render** function you can bind `strings`, `numbers` and `boolean` values as parameters.
```

Comment:
Some suggestions:

In the **Render** function you can bind `string`, `number`and `boolean` values as simple HTML attributes. Just as you do with a normal HTML tag.

In the **componentDidMount** you can setup event listeners and bind attributes that are `arrays` or `objects`. Those type can't be passed as simple HTML attributes so you need to hook them up here.

Use **componentDidUpdate** to update the bind properties values.